client/apps/employee/src/components/TimeEntryList.tsx

```
@@ -102,76 +146,136 @@ export const TimeEntryList: React.FC<TimeEntryListProps> = ({
         );
     }
 
+    // Render status icon
+    const renderStatusIcon = (icon: 'check' | 'warning' | 'x' | 'alert' | 'none', color: string) => {
+        if (icon === 'none') return null;
+
+        if (icon === 'check') {
+            return (
+                <span className={`time-entry-group__status-icon time-entry-group__status-icon--${color}`}>
+                    <svg width="16" height="16" viewBox="0 0 24 24" fill="none">
+                        <circle cx="12" cy="12" r="10" fill="currentColor" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)